pyaedt/hfss.py

```diff
@@ -882,7 +899,24 @@ def create_current_source_from_objects(self, startobj, endobject, axisdir=0, sou
         str
             Name of source created when successful, ``False`` otherwise.
 
+        Examples
+        --------
+
+        Create two boxes that will be used to create a current source
+        named ``'CurrentSource'``.
+
+        >>> box1 = hfss.modeler.primitives.create_box([30, 0, 20], [40, 10, 5],
+        ...                                           "BoxCurrent1", "copper")
+        >>> box2 = hfss.modeler.primitives.create_box([30, 0, 30], [40, 10, 5],
+        ...                                           "BoxCurrent2", "copper")
+        >>> hfss.create_current_source_from_objects("BoxCurrent1", "BoxCurrent2",
+        ...                                                  hfss.AxisDir.XPos,
+        ...                                                  "CurrentSource")
```

Comment:
```suggestion
        >>> hfss.create_current_source_from_objects("BoxCurrent1", "BoxCurrent2",
        ...                                         hfss.AxisDir.XPos,
        ...                                         "CurrentSource")
```
White space suggestion.